### PR TITLE
[WIP] Java time attempt 2

### PR DESCRIPTION
**Pull request by frenchy64**
Second attempt at https://github.com/babashka/babashka/pull/1298. Corresponding PR https://github.com/dm3/clojure.java-time/pull/93

Please answer the following questions and leave the below in as part of your PR.

- [ ] I have read the [developer documentation](https://github.com/babashka/babashka/blob/master/doc/dev.md).

- [ ] This PR corresponds to an [issue with a clear problem statement](https://github.com/babashka/babashka/blob/master/doc/dev.md#start-with-an-issue-before-writing-code).

- [ ] This PR contains a [test](https://github.com/babashka/babashka/blob/master/doc/dev.md#tests) to prevent against future regressions

- [ ] I have updated the [CHANGELOG.md](https://github.com/babashka/babashka/blob/master/CHANGELOG.md) file with a description of the addressed issue.

### CODE REVIEW

File: src/babashka/impl/classes.clj [hunk 1]
```diff
@@ -458,6 +471,7 @@
           java.util.zip.ZipFile
           sun.misc.Signal
           sun.misc.SignalHandler
+          sun.util.calendar.ZoneInfo
```
Comment (borkdude) [inline src/babashka/impl/classes.clj:474]: This seems suspicious. What is the reason for inclusion?

Comment (frenchy64) [reply src/babashka/impl/classes.clj:474]: Yeah I agree. Included it mainly to get tests passing and proceed without further distractions, no other reason. Was planning to have a second pass to cull unnecessary class once the port is one.

**Comment by arichiardi** [comment]
Hi there, can I help in any shape of form here (maybe rebasing?)

I had again the though cause I was about to roll my own conversion functions that `clojure.java-time` has already, if only I could add it to my babashka-compatible project.

**Comment by borkdude** [comment]
There is one comment about the sun class which wasn't addressed.

**Comment by borkdude** [comment]
I'm closing this as this PR looks a bit stale? Feel free to just provide smaller PRs that we can merge immediately. By now several java.time classes have been added, perhaps the PR isn't necessary anymore?

**Comment by arichiardi** [comment]
Sounds good - I'll give it a good second look at some point

**Pull request closed without merging**